Resources/views/CRUD/base_list_field.html.twig

```diff
@@ -11,14 +11,12 @@ file that was distributed with this source code.
 
 <td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}"{% if field_description.options.row_align is defined %} style="text-align:{{ field_description.options.row_align }}"{% endif %}>
     {% set route = field_description.options.route.name|default(null) %}
-    {% set action = route == 'show' ? 'VIEW' : route|upper %}
```

Comment:
We could keep the line for BC compatibility (soft BC break?). Are you extending some templates? Could you find the usage of the var: `action` on our templates (related to this one)?